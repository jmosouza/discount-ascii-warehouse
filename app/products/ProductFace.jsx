import React from 'react';
import PropTypes from 'prop-types';

const ProductFace = ({ size, face }) => (
  <p style={{ fontSize: size }}>{face}</p>
);

ProductFace.propTypes = {
  size: PropTypes.number.isRequired,
  face: PropTypes.string.isRequired,
};

export default ProductFace;
