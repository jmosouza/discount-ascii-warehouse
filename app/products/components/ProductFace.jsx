import React from 'react';
import PropTypes from 'prop-types';

const ProductFace = ({ size, face }) => (
  <span style={{ fontSize: size }}>{face}</span>
);

ProductFace.propTypes = {
  size: PropTypes.number.isRequired,
  face: PropTypes.string.isRequired,
};

export default ProductFace;
