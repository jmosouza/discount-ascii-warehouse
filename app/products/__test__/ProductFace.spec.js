/* global test, expect */
import renderer from 'react-test-renderer';
import React from 'react';
import ProductFace from '../ProductFace';

test('ProductFace', () => {
  expect(renderer.create(
    <ProductFace
      face="¯_(ツ)_/¯"
      size={12}
    />,
  )).toMatchSnapshot();
});
