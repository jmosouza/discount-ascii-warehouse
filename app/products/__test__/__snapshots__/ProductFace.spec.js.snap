// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductFace 1`] = `
<p
  style={
    Object {
      "fontSize": 12,
    }
  }
>
  ¯_(ツ)_/¯
</p>
`;
